requirements.txt

```diff
@@ -25,9 +25,10 @@ tenacity==8.0.1
 testimony==2.2.0
 unittest2==1.1.0
 wait-for==1.2.0
-wrapanapi==3.5.11
```

Comment:
placeholder comment, resolve when new wrapanapi release with fix https://github.com/RedHatQE/wrapanapi/pull/451 is replaced here